### PR TITLE
My very own qmk firmware for a levinson rev 2 using NEO2 keymaps. Will be definitely expanded

### PR DESCRIPTION
**Pull request by Iarumas**
## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Iarumas** [comment]
As far, I'm not really using the project's coding styleguide, but I am willing to change my edits accordingly

**Comment by Iarumas** [comment]
I didn't read and thus follow the coding stylelines until now, but I will change my adds accordingly

File: keyboards/levinson/keymaps/iarumas/neo2-common.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#ifndef NEO2_COMMON_H
+#define NEO2_COMMON_H
+
+
+/*
+This file holds some commen NQN definitions
+*/
+
+
+#define _______ KC_TRNS
```
Comment (noroadsleft) [inline keyboards/levinson/keymaps/iarumas/neo2-common.h]: Lines 10 and 11 here are already defined in the QMK core automatically, so these definitions are redundant here.

https://github.com/qmk/qmk_firmware/blob/7c63c0a95075f6fb4dd94d020ceb5816e6bcca2c/quantum/quantum_keycodes.h#L29-L31

That being the case, you could refactor your headers to remove this entire file and be no worse off.

Same may be possible for `users/iarumas/nqn-common.h` for the same reason, but as I have no experience with the userspace feature, I'm not 100% on that.

File: keyboards/levinson/keymaps/iarumas/neo2-basic-layout.h [hunk 2]
```diff
@@ -0,0 +1,131 @@
+#ifndef NEO2_BASIC_LAYOUT_H
```
Comment (drashna) [inline keyboards/levinson/keymaps/iarumas/neo2-basic-layout.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/levinson/keymaps/iarumas/neo2-common.h [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#ifndef NEO2_COMMON_H
```
Comment (drashna) [inline keyboards/levinson/keymaps/iarumas/neo2-common.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/levinson/keymaps/iarumas/neo2-keys-on-quertz-de-latin1.h [hunk 4]
```diff
@@ -0,0 +1,73 @@
+#ifndef NEO2_KEYS_ON_QUERTZ_DE_LATIN1_H
```
Comment (drashna) [inline keyboards/levinson/keymaps/iarumas/neo2-keys-on-quertz-de-latin1.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/levinson/keymaps/iarumas/neo2-basic-layout.h (hunk 2 above)
Comment (drashna) [reply keyboards/levinson/keymaps/iarumas/neo2-basic-layout.h]: Also, if these match the files in your userspace, then you can actually remove these and use the include to add those.

**Comment by drashna** [comment]
Oh, fun "unknown repo".  Has been deleted. Won't be update, most likely. 

Closing.

**Pull request closed without merging**